types/forest-express-sequelize/forest-express-sequelize-tests.ts

```diff
@@ -5,6 +5,7 @@ import {
     PUBLIC_ROUTES, RecordCreator,
     RecordGetter, RecordRemover, RecordsCounter, RecordSerializer,
     RecordsGetter, RecordsRemover, RecordUpdater, StatSerialized, StatSerializer,
+    SmartActionOption, SmartFieldOption, SmartSegmentOption,
```

Comment:
I would rather write `SmartActionOptions, SmartFieldOptions, SmartSegmentOptions` to be consistent with `CollectionOptions`.